### PR TITLE
AzureCommunicationCalling SDK version 2.0.1-beta.1 release

### PR DESCRIPTION
**Pull request by raosanat**

### CODE REVIEW

File: sdk/communication/AzureCommunicationCalling/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 2.0.1-beta.1 (2021-09-09)
+
+### Bugs Fixed
+- When trying to accept a terminated call `accept` API call will fail with appropriate error.
```
Comment (tjprescott) [inline sdk/communication/AzureCommunicationCalling/CHANGELOG.md]: Recommend:
- `CallingClient.accept` will now throw errors when trying to accept a terminated call or if already in an active call.

File: sdk/communication/AzureCommunicationCalling/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+- SDK rebuilt with latest available AzureCommunicationCommon `1.0.2` ([#344](https://github.com/Azure/Communication/issues/344)).
+- Using `CallKit` implementation in SDK , accepting another call when already in an active call failed.
```
Comment (tjprescott) [inline sdk/communication/AzureCommunicationCalling/CHANGELOG.md:8]: This technical detail isn't relevant to users. Would recommend folding into the above.

Comment (raosanat) [reply sdk/communication/AzureCommunicationCalling/CHANGELOG.md:8]: They are 2 completely different bugs

Comment (raosanat) [reply sdk/communication/AzureCommunicationCalling/CHANGELOG.md:8]: I will try to change the text

**Review by tjprescott — APPROVED** [review]

**Pull request merged**